app/controllers/api/v1/short_form_controller.rb

```diff
@@ -50,17 +50,6 @@ def application_api_params
             :primaryApplicantContact,
```

Comment:
@akegan copying my comment from slack here to better track on PR: "in the lease controller PR, it looks like we still call `submit_lease` in the short form controller’s submit action - is that intended? my impression was that we were replacing that with an explicit call from react to save the lease"